src/applications/vaos/manifest.json

```diff
@@ -3,5 +3,8 @@
   "entryFile": "./vaos-entry",
   "entryName": "vaos",
   "rootUrl": "/vaos",
-  "production": false
-}
\ No newline at end of file
+  "template": {
+    "vagovprod": false,
+    "layout": "page-react.html"
+  }
+}
```

Comment:
This can be confirmed by checking build output for a staging build - `Creating temp React landing page: /vaos`.

http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/master/5106/pipeline/93